maxim/logger/llamaindex/client.py

```
@@ -0,0 +1,299 @@
+import ast
+import time
+
+from llama_index.core.agent.workflow import (
+    AgentWorkflow,
+    AgentOutput,
+    FunctionAgent,
+    ToolCall,
+    ToolCallResult,
+    AgentInput,
+    AgentStream,
+    AgentSetup
```

Ruff (0.12.2)

`llama_index.core.agent.workflow.AgentSetup` imported but unused

Remove unused import: `llama_index.core.agent.workflow.AgentSetup`

(F401)